core/trino-main/src/main/java/io/trino/sql/planner/OptimizerConfig.java

```diff
@@ -756,4 +758,16 @@ public OptimizerConfig setUseExactPartitioning(boolean useExactPartitioning)
         this.useExactPartitioning = useExactPartitioning;
         return this;
     }
+
+    public boolean isForceFixedDistributionForPartitionedOutputOperatorEnabled()
+    {
+        return forceFixedDistributionForPartitionedOutputOperatorEnabled;
+    }
+
+    @Config("experimental.force-fixed-distribution-for-partitioned-output-operator-enabled")
```

Comment:
nit: this is super long. Alsy why `experimental`
maybe `optimizer.force-partitioned-output-fixed-distribution`

Reply:
Also add some documentation here and maybe in some `.rst` if we have any.

Reply:
I intend to remove this property after there's enough confidence that no regression is introduced. End users shouldn't be changing this property, as the plan shape alterations may result in some planner rules (such as `AddExchangesBelowPartialAggregationOverGroupIdRuleSet` to no longer kick in)